local/client/conn/conn.go

```diff
@@ -0,0 +1,66 @@
+package conn
+
+import (
+	"github.com/scootdev/scoot/local/protocol"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+	"log"
+	"net"
+	"time"
+)
+
+// A Dialer can dial a connection to the Scoot server.
```

Comment:
nit: how do we want to do block comments style wise.  I've been doing /* comment */  Looks like standard Go uses //